.gitignore

```diff
@@ -136,3 +136,5 @@ dmypy.json
 
 # talisman
 .talismanrc
+
+**/.DS_Store
```

Comment:
This should be in the system's `.gitignore` file instead of every repo: https://stackoverflow.com/a/7335487